README.md

```diff
@@ -101,3 +101,17 @@ make catalog-build catalog-push
 ```
 
 The Index image powers the listing of the Operator on OperatorHub.
+
+
+## Differences between the GitOps Operator and ArgoCD community Operator 
+
+| Feature | GitOps Operator | Argo CD Community Operator |
+| -------- | -------- | -------- |
+| Out of the box Argo CD Instance     |  ✅    | ❌     |
+| Certified by Red Hat     |   ✅     |   ❌   |
+| Red Hat Productized components     |   ✅     |   ❌   |
+| OpenShift Cluster Configuration using Argo CD    |   ✅     |   OpenShift package in community operator is not bundled properly. This can also be considered as a bug.   |
```

Comment:
The fix is not to include the openshift package -- we're going to move the contents of the openshift package into the gitops-operator.  This used to work without the openshift package being included.

Reply:
ok, I am not sure how it works as the logic is in openshift package.  I will remove the statement 
```
OpenShift package in community operator is not bundled properly. This can also be considered as a bug.
```